securedrop/journalist_app/__init__.py

```diff
@@ -56,11 +56,10 @@ def create_app(config: 'SDConfig') -> Flask:
     def _url_exists(u: str) -> bool:
```

Comment:
LGTM says this is unused :thinking:  https://lgtm.com/projects/g/freedomofpress/securedrop/rev/pr-8b52d687b77d537722116f8be6fbdb9b1574604c

Reply:
good catch, i see `LGTM analysis: Python Successful in 8m — 1 new alert ` in the checks output below but overlooked this because everything looks green